### ISSUE TITLE
installation?

### ISSUE THREAD

**Issue by harry747400** [custom-cards/bar-card#146]
how do i install this? 
i put this:
`- url: /hacsfiles/bar-card/bar-card.js
  type: module` 
into my configuration file, but that got me nothing but errors. i tried to use HACS but HACS can't find it. do i have to add a repository? if so, what's the URL?

**Comment by Sh1bbyz** [comment]
I agree that the installation instructions leave too much to your imagination and it is just sad that no one has answered. I will answer if you still struggle. Maybe it will help someone else to.

I looked into [HA instructions](https://www.home-assistant.io/integrations/frontend/) and this  is how I solved it by typing this into configuration.yaml:

`frontend:
  extra_module_url: 
    - /hacsfiles/bar-card/bar-card.js`
